src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -80,7 +80,14 @@ EmberAfStatus writeFabricAttribute(uint8_t * buffer, int32_t index = -1)
                                     index + 1);
 }
 
-EmberAfStatus writeFabric(FabricId fabricId, NodeId nodeId, uint16_t vendorId, const uint8_t * fabricLabel, int32_t index)
+EmberAfStatus writeCommissionedFabricsAttribute(uint8_t numberCommissionedFabrics)
+{
+
+    return emberAfWriteServerAttribute(0, ZCL_OPERATIONAL_CREDENTIALS_CLUSTER_ID, ZCL_COMMISSIONED_FABRICS_ATTRIBUTE_ID,
```

Comment:
Consider using accessor from Accessors.h.

Reply:
I'll update the PR.